static/src/javascripts/projects/common/utils/config.js

```diff
@@ -19,6 +19,10 @@ define([
         config.page.adUnit = ['/', config.page.dfpAccountId, '/', adUnitOverride].join('');
     }
 
+    config.ophan.pageViewId = new Date().getTime().toString(36) + 'xxxxxxxxxxxx'.replace(/x/g, function() {
+                                    return Math.floor(Math.random() * 36).toString(36);
+                                  });
```

Comment:
since this is non-deterministic, I'm assuming you're not expecting this to match any other id generated elsewhere. However, did you/someone consider using a UUID instead of a custom one?